librz/asm/arch/include/mybfd.h

```diff
@@ -4535,15 +4535,15 @@ bfd_boolean generic_core_file_matches_executable_p(bfd *core_bfd, bfd *exec_bfd)
 #ifdef DEBUG_BFD_SEND
 #undef BFD_SEND
 #define BFD_SEND(bfd, message, arglist) \
-	(((bfd) && (bfd)->xvec && (bfd)->xvec->message) ? ((*((bfd)->xvec->message))arglist) : (bfd_assert(__FILE__, __LINE__), NULL))
+	(((bfd) && (bfd)->xvec && (bfd)->xvec->message) ? ((*((bfd)->xvec->message))arglist) : (bfd_rz_return_if_fail(__FILE__, __LINE__), NULL))
```

Comment:
This seems to be a result of the `sed`, unnecessary change

Reply:
Okay so I shouldn't change this?
Sorry I actually got confused. Please is there a way I could know where to place the changes and where not to ?

Reply:
Yes, you could see it's not a pure `assert()` but `bfd_assert()`